src/api/utils/email.py

```diff
@@ -57,3 +62,26 @@ def contact_project_admins(project, subject, body, from_email):
     # lots of checking/validation/antispam here
     # email_project_admins(project, subject, body, from_email)
     pass
+
+
+@run_in_thread
```

Comment:
`run_in_thread` is cool, but should we be sending email synchronously and letting user know when the new profile was added but the email didn't successfully send?